src/setup/devnet/templates/remote/bor-service-host.sh

```diff
@@ -27,6 +29,29 @@ cat > ganache.service <<EOF
     KillSignal=SIGINT
 EOF
 
+if [ "$FLAG" = "config" ]
+then
+cat > bor.service <<EOF
+[Unit]
+  Description=bor
+  StartLimitIntervalSec=500
+  StartLimitBurst=5
+[Service]
+  Restart=on-failure
+  RestartSec=5s
+  WorkingDirectory=$NODE_DIR
+  Environment=PATH=$PATH
+  EnvironmentFile=$HOME/metadata
+  #ExecStartPre=/bin/bash $NODE_DIR/bor-setup.sh
+  ExecStart=/bin/bash $NODE_DIR/bor-start-config.sh
```

Comment:
This is the only difference between config enabled/disabled, right?
Thinking if we can improve it somehow

Reply:
Yes, that is correct.
I agree, open to suggestions.

Reply:
It would require some manipulations with `sed` which would be harder to maintain. Let's keep it as it is, thanks.

Reply:
I agree, thanks!